app/components/candidate_interface/carry_over_interstitial_component.html.erb

```diff
@@ -13,6 +13,14 @@
       Continue your application to apply for courses starting in the <%= next_academic_cycle %> academic year instead.
     </p>
 
+    <p class="govuk-body">
+      You no longer need to request and receive references before submitting your application.
+    </p>
+
+    <p class="govuk-body">
+      Instead, you’ll need to give details of 2 people who can give references if you accept a place on a course.
+    </p>
```

Comment:
We could maybe wrap this in a condition so that it only appears for this particularly rollover year? Or otherwise we can just remove it next year...